mlrun/model_monitoring/applications/histogram_data_drift.py

```diff
@@ -189,21 +189,24 @@ def _get_general_drift_result(
         )
 
         status = self._value_classifier.value_to_status(value)
+
+        model_endpoint_stats = {
+            EventFieldType.CURRENT_STATS: json.dumps(
+                monitoring_context.sample_df_stats
+            ),
+            EventFieldType.DRIFT_MEASURES: json.dumps(
+                metrics_per_feature.T.to_dict()
+                | {metric.name: metric.value for metric in metrics}
```

Comment:
If I understand correctly, this line represents the mean value for each metric. I think you can remove it here, as we’re already creating MM metrics for these values.